### PR TITLE
Fuel angular momentum is lost as-is only from SRMs

### PR DESCRIPTION
**Pull request by eggrobin**
Other engines should also lose some angular momentum, but since liquid fuels go through pipes and injectors they should lose some of their angular momentum before they leave the rocket. This will be addressed in a future pull request.

### CODE REVIEW

File: base/zfp_compressor.cpp [hunk 1]
```diff
@@ -15,7 +15,7 @@ namespace zfp_compressor_internal {
 // small for our purposes.  Therefore, we must not include the bit
 // ZFP_HEADER_META.  We still need a header, though, to record the compression
 // parameters.
-static constexpr uint header_mask = ZFP_HEADER_MODE | ZFP_HEADER_MAGIC;
+static constexpr uint header_mask = ZFP_HEADER_FULL;
```
Comment (pleroy) [inline base/zfp_compressor.cpp]: Please revert this change.

File: ksp_plugin/part.hpp [hunk 2]
```diff
@@ -74,6 +74,10 @@ class Part final {
   Mass const& mass() const;
   void set_inertia_tensor(InertiaTensor<RigidPart> const& inertia_tensor);
   InertiaTensor<RigidPart> const& inertia_tensor() const;
+  // Whether this part is a solid rocket motor, whose lost mass is expelled with
```
Comment (pleroy) [inline ksp_plugin/part.hpp]: ... the lost mass of which ...

Comment (eggrobin) [reply ksp_plugin/part.hpp]: As discussed, this is fine.
